### DISCUSSION TITLE
Add another DNS IP on SO Manager

### DISCUSSION THREAD

**Discussion by jozb09** [Security-Onion-Solutions/securityonion#9192] [General] [answered]
Hello team,

I managed to install manager, search, and forward as distributed. But my issue is I want to add another DNS IP of our internal one to all nodes because during the installation I only put one internal DNS IP. 

Can someone help me with which config file I should change?

**Comment by dougburks** [comment, marked as answer]
Take a look at `/etc/resolv.conf`. Depending on how you installed, it may tell you to reference another location.
